### PR TITLE
Remove ingestion stats join and other changes

### PR DESCRIPTION
**Pull request by smritinahata305**
https://screencast.googleplex.com/cast/NTM2NjEwMzA0MjY4Njk3NnxlZWMzZjIyZS05ZQ
Default dashboards screencast

### CODE REVIEW

File: views/refinements/udm_events_aggregates.view.lkml
Comment (pradeepthangamuthu) [inline views/refinements/udm_events_aggregates.view.lkml:1]: Is this change needed ?

File: views/custom/ingestion_metric_with_ingestion_stats.view.lkml
Comment (pradeepthangamuthu) [inline views/custom/ingestion_metric_with_ingestion_stats.view.lkml:1]: This is a breaking change, like for total_entry_number the sql will be formed with entry_number and log_count, but we won't have entry_number in the new SQL. 

For the fields which are part of ingestion_stats and not part of ingestion_metrics assign a default value, like `null as entry_number`. So the SQL won't break.

File: views/refinements/udm_events_aggregates.view.lkml
Comment (smritinahata305) [reply views/refinements/udm_events_aggregates.view.lkml:1]: Yes, it is used in default dashboards

File: views/custom/ingestion_metric_with_ingestion_stats.view.lkml [hunk 1]
```diff
@@ -9,23 +9,67 @@ view: ingestion_metric_with_ingestion_stats {
...
+
+  dimension: size_bytes {
+    description: "Deprecated"
+    type: number
+    sql: null ;;
+  }
+
+  dimension: time_bucket {
+    description: "Deprecated"
+    type: string
```
Comment (pradeepthangamuthu) [inline views/custom/ingestion_metric_with_ingestion_stats.view.lkml:57]: Is this string or number?

**Review by pradeepthangamuthu — APPROVED** [review]
